### ISSUE TITLE
Create User Profile HTML mockup

### ISSUE THREAD

**Issue by brylie** [meteorcrowd/crowducate-next#148]

**Comment by Rahbaran** [comment]
here are two different version of profile mock up .1. Feedback welcomed. One is separated into tabs (just uploading the first tab), the other more "blurred". The long, blurred version might look too long but don't forget that the mockup shows a very active user + the image doesn't show any scrolling.

## Long Version:

![profile_cu_1](https://cloud.githubusercontent.com/assets/3035315/7770049/55a66d34-008a-11e5-84ed-038faa7cf0ce.png)

## Short Version:

![profile_cu_2](https://cloud.githubusercontent.com/assets/3035315/7770048/55a5c834-008a-11e5-9d6c-ab5c03759eed.png)

**Comment by sebbel** [comment]
I really like it.

**Comment by brylie** [comment]
Very nice. Perhaps the short version can indicate the user role in each class (e.g. student, contributor, teacher, etc.).

**Comment by Rahbaran** [comment]
Heureka, i have a new idea. Let's use drop down (default: latest activity). This makes everything cleaner than apps.

![profile_dropdown](https://cloud.githubusercontent.com/assets/3035315/7800823/cb6f3ae4-031b-11e5-9f69-d80b51dd4833.png)
